### ISSUE TITLE
config file: throw-space-n

### ISSUE THREAD

**Issue by alexgenaud** [ianyh/Amethyst#1521]
"Move the currently focused window to the n-th space" (throw-space-< screen-number > ) works for me configured through the GUI, but not when loaded from the ~/.amethyst.yml file.

It's likely that I just don't know how to configure n-th spaces (ten Desktops).
All of my ~/.amethyst.yml configurations seem to load and work correctly,
exception for throw-space-< screen-number >.

I've tried:

```
  throw-space-i:
  mod: mod3
  key: i
```

```
  throw-space-n:
  mod: mod3
  key: n
```

And copied 10 variations with the number in the action name and key:

```
  throw-space-7:
  mod: mod3
  key: 7
```

My mod3 is defined (Shift-Control) and works with left-right arrows:

```
mod3:
  # - option
  - shift
  - control
  # - command
```

I've tried with 0.19 and 0.20.0b3

**Comment by ianyh** [comment]
Could you upload your whole config file?

**Comment by alexgenaud** [comment]
Certainly. The 10 Desktops are probably a mess in the config, but I'd be happy to try/test different settings.

<img width="722" alt="image" src="https://github.com/ianyh/Amethyst/assets/32287/9a316ceb-8035-4242-901d-68055c67ca2b">

Note: mod-N "Desktop" should probably say "Screen" instead.

^ 1-9,0 are set in Mission Control. I do not believe the shortcuts can be altered. Thus ^-Shift seem the natural obvious shortcut to move focused window to Desktop N.

# Feature Request

Amethyst has a setting to "follow window" when thrown to another desktop (or not). Actually, I would prefer BOTH.

Most often I only want 1, 2, or 3 windows per Desktop/Space. I might shift view to another Desktop temporarily (to review documentation or check email or change music) and return to my "work" (probably Desktop 1). If not just shifting views, the most intuitive experience is to "follow the thrown window" especially with multiple screens (which can get confusing).

However, sometimes I want to move a temp window into my "work" Desktop and then move it ("get rid of it" without following). Sometimes a Desktop just gets cluttered, and I want to throw windows to various Desktops (often "anywhere but here").

@ianyh I would suggest that "viewing" Desktops/Spaces (via Mission Control) is Ctrl only. Throwing windows to another Desktop (and not following) would be Ctrl-Opt. And Throw-and-follow would be Ctrl-Opt-Shift.

Naturally, I believe the config above is most logical. I expect many others would find "my rules" easiest to remember and use, too. All combinations use the left mod-keys and ASCII on the right. All shortcuts include Ctrl and shortcuts that remain on a single Desktop/Screen include Option (whereas Mission Control's Ctrl-(1-9,0) and Ctrl-arrow are Ctrl only). Shift "moves" windows and corresponds with its non-shifted "view" shortcut.

**Comment by alexgenaud** [comment]
@ianyh renamed to keep GH happy
[dot_amethyst_yml.txt](https://github.com/ianyh/Amethyst/files/11975263/dot_amethyst_yml.txt)

**Comment by mtreca** [comment]
I am running in the same issue when setting mod3 as ctrl+shift.

**Comment by daibertdiego** [comment]
Same issue here with:

`mod3:
  - option
  - shift
  - control

throw-space-1:
  mod: mod3
  key: 1

throw-space-2:
  mod: mod3
  key: 2

throw-space-3:
  mod: mod3
  key: 3

throw-space-4:
  mod: mod3
  key: 4

throw-space-5:
  mod: mod3
  key: 5

throw-space-6:
  mod: mod3
  key: 6

throw-space-7:
  mod: mod3
  key: 7

throw-space-8:
  mod: mod3
  key: 8

throw-space-9:
  mod: mod3
  key: 9

throw-space-10:
  mod: mod3
  key: 0`

And this is only working with the config file. Before, this feature was working as expected.

**Comment by hyogoa2c** [comment]
I have similar issue with config:
```
mod1:
  - option
  - shift

....
....
throw-space-1:
  mod: mod1
  key: 1
throw-space-2:
  mod: mod1
  key: 2
throw-space-3:
  mod: mod1
  key: 3
throw-space-4:
  mod: mod1
  key: 4
throw-space-5:
  mod: mod1
  key: 5
throw-space-6:
  mod: mod1
  key: 6
throw-space-7:
  mod: mod1
  key: 7
throw-space-8:
  mod: mod1
  key: 8
throw-space-9:
  mod: mod1
  key: 9
```

**Comment by jc65536** [comment]
The fix is to put single quotes around your numbers.
